api/libretimeapi/views.py

```diff
@@ -141,9 +142,19 @@ class PreferenceViewSet(viewsets.ModelViewSet):
 class ScheduleViewSet(viewsets.ModelViewSet):
     queryset = Schedule.objects.all()
     serializer_class = ScheduleSerializer
-    filter_fields = ("starts", "ends", "playout_status", "broadcasted", "is_valid")
+    filter_fields = ("starts", "ends", "playout_status", "broadcasted")
     model_permission_name = "schedule"
 
+    def get_queryset(self):
+        filter_valid = self.request.query_params.get("is_valid")
+        if filter_valid is None:
+            return self.queryset.all()
+        filter_valid = filter_valid.strip().lower() in ["true", "yes", "1"]
```

Comment:
Small improvement, using a tuple will improve performances, not really a problem though, but we want to reach perfection here !

Reply:
Good point, will do that